src/Compute/Compute/ChangeLog.md

```diff
@@ -42,6 +42,10 @@
     - SinglePlacementGroup must be false. Throws an error if true. 
     - Networking Profile's API version is 2020-11-01 or later.
     - Networking Profile IP Configurations Primary property is set to true.
+* Contains updates to the following powershell cmdlets
+	- `SetAzVmssDiskEncryptionExtension` : Added extension parameters for the cmdlet to work with test extensions and parameter `EncryptFormatAll` for Virtual Machine Scale Sets
```

Comment:
```suggestion
    - `SetAzVmssDiskEncryptionExtension` : Added extension parameters for the cmdlet to work with test extensions and parameter `EncryptFormatAll` for Virtual Machine Scale Sets
```
Please use spaces instead of tabs for consistency

Reply:
Also, changelog.md has been updated on main branch recently. Please sync with main and make sure the lines you added appear just below the "Upcoming release" title.